.babelrc

```diff
@@ -1,5 +1,6 @@
 {
     "presets": [
+        "env",        
         ["es2015", {"modules": false}],
```

Comment:
We can then remove this es2015 preset, both in `.babelrc` and `npm rm` it

Reply:
Sounds good. Will do that.